media/css/firefox/firstrun/firstrun-quantum.scss

```diff
@@ -33,11 +34,14 @@ $mq-large:  'screen and (min-width: 860px)';
 }
 
 body {
+    background: #003eaa;
     margin: 0;
-    overflow: hidden;
-
-    @media (max-height: 710px) {
-        overflow-y: auto;
+    overflow-y: auto;
+    @media #{unquote($mq-medium)} {
```

Comment:
I don't think you need these `unquote`'s?

Reply:
TIL.